bynder_local.module

```diff
@@ -19,75 +24,94 @@ use GuzzleHttp\Exception\ConnectException;
 function bynder_local_media_presave(MediaInterface $media) {
   $source = $media->getSource();
 
-  if ($source instanceof Bynder && $media->hasField('field_bynder_image') && !$media->get('field_bynder_image')->entity) {
+  $config = Drupal::config('bynder_local.settings');
+  $local_field = $config->get('local_image_field');
```

Comment:
Please, add a default value to avoid NULL.

Reply:
I've approached this in a different way; the value is now set via the module config on install, so we can always rely on it being set.